misk-crypto/README.md

```diff
@@ -0,0 +1,59 @@
+This module provides Misk with the ability to store and manage cryptographic key material in a
```

Comment:
really nice docs, thank you!

Reply:
let's chat in #misk but it would be great if you could create a crypto-exemplar service in staging that demonstrated the functionality out. We've been adding exemplar services that demonstrate various platform offerings as an easy way for folks to view/reuse example code.

Reply:
Good point!
That's actually what I'm working on right now, I want to be sure I can easily create and deploy an app that uses this module before I merge this PR.